.github/workflows/pull-request-checks.yaml

```diff
@@ -375,6 +375,7 @@ jobs:
         run: | 
           sudo apt-get update
           sudo apt-get install --no-install-recommends -y g++ flex bison cmake ninja-build maven jq libxml2-utils dpkg-dev ccache
+          sudo apt-get --purge remove -s libgcc-s1
```

Comment:
Huh, that seems screwy? So the github actions ubuntu-18.04 container isn't necessarily actually ubuntu-18.04?